### PR TITLE
Removes min-height from action page photos

### PR DESCRIPTION
**Pull request by lkpttn**
This PR just removes min-height from action steps photos. 

This is the easiest way to handle the image stretching on mobile and desktop sizes. Later on, we should make these into squares with proper overflows and such.

### CODE REVIEW

File: resources/assets/components/ActionPage/actionPage.scss [hunk 1]
```diff
@@ -62,7 +62,7 @@
       padding: $base-spacing / 2;
 
       img {
-        min-height: 200px;
+/*         min-height: 200px; */
```
Comment (deadlybutter) [inline resources/assets/components/ActionPage/actionPage.scss]: I would just delete the entire selector

Comment (weerd) [inline resources/assets/components/ActionPage/actionPage.scss]: ya, don't comment it out if removing it. just delete it! 🗑

Comment (lkpttn) [inline resources/assets/components/ActionPage/actionPage.scss]: Do we want to keep if we know we'll need to target it again when we actually fix this?

Comment (deadlybutter) [inline resources/assets/components/ActionPage/actionPage.scss]: @lkpttn Leave a `TODO: ...`

**Pull request merged**